parser/parser_test.go

```diff
@@ -30,7 +31,12 @@ func TestBasicValidation(t *testing.T) {
 	assert.NoError(t, err)
 	for _, file := range files {
 		if !file.IsDir() {
+			if file.Name() == "paymentconfirmation.json" {
+				fmt.Printf("done")
+			}
```

Comment:
Why do we need this?

Reply:
aha，it's the test code and i forget to remove it ^^